src/main/java/build/buildfarm/operations/finder/EnrichedOperationBuilder.java

```diff
@@ -98,13 +114,15 @@ private static Operation jsonToOperation(String json) {
             .ignoringUnknownFields();
 
     if (json == null) {
+      logger.log(Level.SEVERE, "Operation Json is empty");
       return null;
     }
     try {
       Operation.Builder operationBuilder = Operation.newBuilder();
       operationParser.merge(json, operationBuilder);
       return operationBuilder.build();
     } catch (InvalidProtocolBufferException e) {
+      logger.log(Level.SEVERE, e.getMessage());
```

Comment:
We should just log "some error message", e here vs e.getMessage(). Same with below ones as well.
Ex: logger.log(Level.SEVERE, "InvalidProtocolBufferException while building an operation.", e);

Reply:
yeah, that's better.  switched.